### ISSUE TITLE
Parse error on any token inside a function

### ISSUE THREAD

**Issue by MaxMilton** [sveltejs/eslint-plugin-svelte3#2]
When parsing a function inside a svelte component `<script>` block, a `ParseError` is thrown on the first token inside the function:

![repro1-func](https://user-images.githubusercontent.com/14946546/52904501-81b7ff00-3281-11e9-8324-a7b3ede13310.png)

Interestingly, it also throws then the function is commented out:

![repro2-commented](https://user-images.githubusercontent.com/14946546/52904530-92687500-3281-11e9-9de6-28cdb261188d.png)

Same thing for arrow functions:

![repro3-arrow-func](https://user-images.githubusercontent.com/14946546/52904532-9bf1dd00-3281-11e9-971b-ba0b8ab91505.png)

**Comment by MaxMilton** [comment]
After a little debugging I discovered these errors are actually being thrown by `eslint-plugin-html` with:

```js
  settings: {
    'html/html-extensions': ['.html', '.svelte'],
  }
```

Removing that settings line fixes those parse errors. Might be worth adding a note about this to the readme.

**Comment by Conduitry** [comment]
There is a line in the readme "Also make sure you do not have `eslint-plugin-html` enabled on the files you want linted as Svelte components, as the two plugins won't get along." but maybe this should be put in some more prominent location?

**Comment by MaxMilton** [comment]
Sorry @Conduitry , you're totally right! User error :grimacing: 

I did read through the readme when I ran into the issue but somehow I missed that sentence. Probably because it's under the "CLI" heading.

Somewhere more prominent would be ideal or even adding `**Important**: `.

**Comment by Conduitry** [comment]
:+1: Moved the eslint-plugin-html note to a more prominent spot, and added an 'important' label.
